src/RenderWebGL.js

```diff
@@ -737,6 +737,40 @@ class RenderWebGL extends EventEmitter {
     }
 
     /**
+     * Update the position object's x & y members to keep the drawable fenced in view.
+     * @param {int} drawableID - The ID of the Drawable to update.
+     * @param {object} position - An object with x & y members to be fenced
+     *                          - These values are updated in the object to become fenced
+     */
+    getFencedPositionOfDrawable (drawableID, position) {
```

Comment:
By our convention, a function whose name starts with `get` is usually intended to retrieve or calculate a value, often with few or no caller-visible side effects. Since this one doesn't return anything and is entirely about side effects, I think it would make sense to change the name. Maybe `setFencedPositionOfDrawable`?